com.github.mtkennerly.ludusavi.yaml

```diff
@@ -0,0 +1,48 @@
+app-id: com.github.mtkennerly.ludusavi
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-stable
+build-options:
+  append-path: /usr/lib/sdk/rust-stable/bin
+  env:
+    CARGO_HOME: /run/build/ludusavi/cargo
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
```

Comment:
What is network access needed for?

Reply:
The app automaticly syncs manifests of where the saves are located on launch.